include/envoy/server/admin.h

```diff
@@ -65,7 +65,7 @@ class AdminStream {
  */
 #define MAKE_ADMIN_HANDLER(X)                                                                      \
   [this](absl::string_view path_and_query, Http::ResponseHeaderMap& response_headers,              \
-         Buffer::Instance& data, Server::AdminStream& admin_stream) -> Http::Code {                \
+         Buffer::Chunker& data, Server::AdminStream& admin_stream) -> Http::Code {                 \
```

Comment:
I don't think we need to put this in the general `Buffer` namespace. This can be private to 'admin'.

Reply:
Got it.